packages/microcode/Cargo.toml

```diff
@@ -11,9 +11,9 @@ path = "../packages.rs"
 # Use latest-srpm-urls.sh to get these.
 
```

Comment:
This comment is not accurate anymore, as is `latest-srpm-urls.sh`. I'd drop both for now.